src/windows/com.rs

```diff
@@ -152,6 +156,7 @@ impl COMPort {
             handle: handle as HANDLE,
             timeout: Duration::from_millis(100),
             port_name: None,
+            dtr: false,
```

Comment:
I would do the call to `DeviceIoControl` here since we cannot assume the state. But if the call fails (which I would consider unlikely), the most sane option is to just panic as it means that the handle from FFI did something weird.